spec/requests/api/v1/users_spec.rb

```diff
@@ -73,9 +87,6 @@
           expect(json["data"]["attributes"]["role"]).to eq "basic"
           expect(json["data"]["relationships"]["employer"]["data"]["id"]).to eq organization.id.to_s
           expect(json["data"]["relationships"]["employer"]["data"]["type"]).to eq "organization"
-          expect(json["included"][0]["id"]).to eq organization.id.to_s
```

Comment:
All of these `json["included"][0]`  checks are just invalid now that sessions are in there. It didnt seem to make sense to replace as they're covered in the relationships already.

Reply:
oh yeah, I started replacing those with: 

```
 expect(json["included"]).to include(a_hash_including("type" => "work_process", "id" => oswp.id.to_s))
```

Can you add that in so we at least have a check that data we want in the `included` array is there?